### ISSUE TITLE
RimHUD labels are broken

### ISSUE THREAD

**Issue by TheLonerD** [Dubwise56/Rimatomics#38]
RimHUD labels are broken when Rimatomics mod is activate. Select any pawn, hover to any skill and popup will appear: some text have placeholders instead of actual values.
Game version: 1.4.3580 rev1304
My mod list:
<details><summary>Mod list</summary>
<p>

```xml
<activeMods>
  <li>brrainz.harmony</li>
  <li>ludeon.rimworld</li>
  <li>ludeon.rimworld.royalty</li>
  <li>ludeon.rimworld.ideology</li>
  <li>ludeon.rimworld.biotech</li>
  <li>jaxe.rimhud</li>
  <li>dubwise.rimatomics</li>
</activeMods>
```
</p>
</details>

Screenshots:
<details><summary>With Rimatomics</summary>
<p>

![image](https://user-images.githubusercontent.com/6560567/210741896-e8567ccb-1b3e-4df2-b6dc-3d5c909be0db.png)

</p>
</details>
<details><summary>Without Rimatomics</summary>
<p>

![image](https://user-images.githubusercontent.com/6560567/210742406-1c0e6cec-3ac2-481d-8a45-a27bcfc9f872.png)

</p>
</details>

**Comment by Bluebottle39** [comment]
Hello, I have the same issue but without RimHUD. It looks like its the same popup as in the bio.

**Comment by Diakon5** [comment]
https://github.com/Dubwise56/Rimatomics/blob/ea1eab69ea641612a46ef6951a294ab8ae352b0d/Languages/English/Keyed/Rimatomics.xml#L273

This translation key must be accidentally overriding a vanilla one, no clue where it's used due to a lack of source code.

**Comment by Sindusk** [comment]
> https://github.com/Dubwise56/Rimatomics/blob/ea1eab69ea641612a46ef6951a294ab8ae352b0d/Languages/English/Keyed/Rimatomics.xml#L273
> 
> This translation key must be accidentally overriding a vanilla one, no clue where it's used due to a lack of source code.

This is correct and the SkillLevel key is not translated in any of the code that I could read from decompiling the assembly. This key can be removed.
